### DISCUSSION TITLE
Any plans to upgrade JAXWS/jaxb implementation to jakarata package

### DISCUSSION THREAD

**Discussion by rohitbatta** [OxalisCommunity/oxalis#656] [General]
Are there any plans to upgrade jaxb implementation to 4.x.x using `Jakarta.xml.bind` implementation instead of `javax.xml.bind` implementation?

Oxalis is using JAXWS and JAXB apis which are now removed from JDK since version 11 and moved to Jakarta.

**Comment by aaron-kumar** [comment]
@rohitbatta : It is already part of https://github.com/OxalisCommunity/Oxalis-internal-roadmap/issues (You must be Oxalis Member to access "Oxalis-internal-roadmap")

Converting this to discussion

**Reply by HegeErdal** [reply]
@aaron-kumar : Do you have any timeline for when this is supposed to be implemented?

**Reply by aaron-kumar** [reply]
@HegeErdal  : Probably it will be part of H2, 2024. More clarity will be given in "Oxalis Community Business Meeting" on 20th March 2024

**Reply by smahieu87** [reply]
@aaron-kumar any update regarding the timeline?

**Reply by aaron-kumar** [reply]
H2, 2024 (_work will start end of Q3 and most likely will be available in Q4_). Note that any important Peppol compliance requirement will take priority.
